plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -468,63 +486,22 @@ public static function get_plugin( $slug ) {
 					'</a>'
 				),
 				'is_visible'     => array(
+					self::get_rules_for_wcservices_tax_countries(),
```

Comment:
This entire `is_visible` key controls presence of WCS&T branded as "WooCommerce Tax" in the core profiler/setup wizard.